warehouse/query-core/src/main/java/datawave/query/planner/ThreadedRangeBundlerIterator.java

```diff
@@ -139,7 +139,7 @@ public boolean hasNext() {
                     // amount of time to wait has elapsed OR we have processed all of our ranges before continuing
                     while (this.rangeQueue.size() < numRangesToBuffer && this.rangeQueue.remainingCapacity() > 0
                                     && (startTimeMillis + rangeBufferTimeoutMillis) > System.currentTimeMillis() && !rangeConsumer.isStopped()) {
-                        Thread.sleep(rangeBufferPollMillis);
```

Comment:
wait on the producerLock releases the lock for this thread and allows other threads to enter the synchronized block.  Is that really what we want?

Reply:
Yeah -- not entirely sure.  Changed it back to `sleep()`.

Reply:
The diff is not reflecting this was changed back to the `sleep()` call. There's also a similar change above in `RecordIterator` so if we're not sure the wait() is appropriate, might be worth flipping that back for now too. We can always dig in with a more specific task for those later.

Reply:
reverted